aws/rust-runtime/aws-config/Cargo.toml

```diff
@@ -13,13 +13,13 @@ client-hyper = ["aws-smithy-client/client-hyper"]
 rustls = ["aws-smithy-client/rustls"]
 native-tls = ["aws-smithy-client/native-tls"]
 rt-tokio = ["aws-smithy-async/rt-tokio", "tokio/rt"]
+sso = ["dep:aws-sdk-sso", "dep:ring", "dep:hex", "dep:zeroize"]
 
-default = ["client-hyper", "rustls", "rt-tokio"]
+default = ["client-hyper", "rustls", "rt-tokio", "sso"]
```

Comment:
I've been thinking about feature gating STS & SSO for a while—they're the heaviest parts of aws-config since they pull in 2 full SDK clients. Not saying we shouldn't also swap out ring, but I think disabling SSO is actually a decently useful feature.

Reply:
We might want a clearer name for the sso feature though—maybe `credentials-sso`?

Reply:
In the context of Wasm, I believe the host would provide the credentials most of the time. Taking the browser as an example, you would already have the credentials cached and would be passing to the Wasm module while instantiating it.

Reply:
I believe both need to happen: feature flagging and swapping ring. If you both don't mind, I think we can just rename the feature as suggested and leave the rest for another PR. Is that ok?

Reply:
That's fine.